packages/react-dom/src/__tests__/utils/IntersectionMocks.js

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
Moved these utilities out of ReactDOMTestSelectors-test.js so we can reuse in ReactDOMFragmentRefs-test.js